commands/utility/reminder.js

```diff
@@ -14,13 +14,14 @@ class Reminder extends Command {
     });
   }
 
-  async run(message, args, level) { // eslint-disable-line no-unused-vars
-    if (args.length === 0) {
+  async run(message, [...text]) {
+    text = text.join(" ");
+    if (!text.length) {
```

Comment:
Pro tip: `text` is an empty array, therefore its length is 0, you can join it later when you need it as string.